### ISSUE TITLE
Multi-type Tenants support updated in the Docs (#483) but latest released version (v1.2.0) is behind

### ISSUE THREAD

**Issue by marclneto** [Corvia/django-tenant-users#504]
I've been banging my head against the wall for a while trying to figure out why the new support for multi-type tenants wasn't working, comparing my settings with the docs and the latest commit in master, until I realized the latest release is behind with this feature.

I've compared both commits and it seems the main (functional) diferences happened at utils and tasks:
https://github.com/Corvia/django-tenant-users/compare/594a86b..7728939

Is it by design or am I doing something wrong?

As a workaround, I've installed from the master branch and it worked fine:
```bash
pip install -I -U git+https://github.com/Corvia/django-tenant-users.git@master
```

## Expected Behavior
```bash
[1/1 (100%) standard:testtenant_1700000345] === Starting migration
[1/1 (100%) standard:testtenant_1700000345] Operations to perform:
```

## Actual Behavior
```bash
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/casamentosaas/management/commands/tenant_users_setup.py", line 48, in handle
    fqdn = provision_tenant(test_company_name, "testtenant", test_user_email, tenant_type="type1")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: provision_tenant() got an unexpected keyword argument 'tenant_type'
```

## Possible Fix
Release current build.

## Steps to Reproduce
This is not required, but it would be highly appreciated if you
provided a link to a live example, or an unambiguous set of steps to
reproduce this bug. Include code to reproduce, if relevant.

## Your Environment
* Version used: 1.2.0
* Environment name and version (e.g. Django 3.2.15, pip 9.0.1): Django 4.2.7
* Operating System and version: Docker Desktop 4.25.0 - Windows 11 (Dockerfile FROM python:3.11)
* Link to your project (if applicable): N/A

**Comment by Dresdn** [comment]
To clarify, the feature was added in https://github.com/Corvia/django-tenant-users/pull/475

I'll create a new release

**Comment by Dresdn** [comment]
Released [1.3.0](https://github.com/Corvia/django-tenant-users/releases/tag/1.3.0)
